developer_docs.md

```diff
@@ -79,39 +56,61 @@ git submodule update
 
 #### Create and activate a Python virtual environment
 
+The following instructions assume you have [Python 3](https://www.python.org/downloads/) installed on your machine.
+
 You can use either `pyenv` or `python3-venv`:
 
 ##### pyenv
 
 1. Install [pyenv](https://github.com/pyenv/pyenv). The [pyenv installer](https://github.com/pyenv/pyenv-installer) is recommended.
 1. Create a virtual environment.
-    ```bash
-    pyenv virtualenv rdl-standard
-    ```
+   ```bash
+   pyenv virtualenv rdl-standard
+   ```
 1. Activate the virtual environment
-    ```bash
-    pyenv activate rdl-standard
-    ```
+   ```bash
+   pyenv activate rdl-standard
+   ```
 1. Set the local application-specific virtual environment. Once set, navigating to the `rdl-standard` directory will automatically activate the environment.
-    ```bash
-    pyenv local rdl-standard
-    ```
+   ```bash
+   pyenv local rdl-standard
+   ```
 
 ##### python3-venv
 
-1. Install [python3-venv](https://docs.python.org/3/library/venv.html).
+If you are using Python 3.3 or newer, `venv` is included in the standard Python installation.
 
-    ```bash
-    sudo apt-get install python3-venv
-    ```
-1. Create a virtual environment.
-    ```bash
-    python3 -m venv .ve
-    ```
+1. Install [python3-venv](https://docs.python.org/3/library/venv.html).
+    a. Linux users
+        ```bash
+        sudo apt-get install python3-venv
+        ```
+    b. MacOS users
+        ```bash
+        pip3 install venv
+        ```
+    c. Windows users
+        ```bash
+        py -m pip install venv
+        ```
```

Comment:
[Python versions older than 3.8 are end of life](https://devguide.python.org/versions/#versions) and some dependencies of `./manage.py` require Python 3.7 or greater.

Therefore, can we remove this section and instead update the sentence at the start of 'create and activate a Python virtual environement' to read "The following instructions assume that you have Python 3.8 or newer installed on your machine."?